tutor/templates/build/openedx/Dockerfile

```
@@ -156,7 +156,7 @@ USER ${APP_USER_ID}
 # https://hub.docker.com/r/powerman/dockerize/tags
 COPY --link --from=docker.io/powerman/dockerize:0.19.0 /usr/local/bin/dockerize /usr/local/bin/dockerize
 COPY --chown=app:app --from=edx-platform / /openedx/edx-platform
-COPY --chown=app:app --from=locales /openedx/locale /openedx/locale
+{% if not OPENEDX_ATLAS_PULL %}COPY --chown=app:app --from=locales /openedx/locale /openedx/locale{% endif %}
```

One more revert and it'll be good.

```suggestion
COPY --chown=app:app --from=locales /openedx/locale /openedx/locale
```

---

done